### DISCUSSION TITLE
Request to review/add asdf-kiota

### DISCUSSION THREAD

**Discussion by andreaTP** [asdf-community/.github#57] [General]
Request to add [asdf-kiota](https://github.com/andreaTP/asdf-kiota) to this community.

It is based on [asdf-plugin-template](https://github.com/asdf-vm/asdf-plugin-template).

**Comment by smorimoto** [comment]
@andreaTP The smoothest transfer path is: (It will hopefully be well documented this week!)

1. Submit a PR to create a team for the plugin: Consult the PR https://github.com/asdf-community/infrastructure/pull/13
2. Once it's merged and you're invited into the organisation, transfer the repository.
4. Submit a PR to import the repository and put it under Terraform's control just like 1: https://github.com/asdf-community/infrastructure/pull/14
5. ALL DONE! In some cases you will need to merge Renovate PR etc to keep the GHA up-to-date.

**Reply by andreaTP** [reply]
Thanks for the quick feedback.
Let me go through the process!

**Reply by smorimoto** [reply]
Good morning. (At least from Japan!) Everything's done! Thanks for working with me!
